include/fmt/format.h

```diff
@@ -862,7 +862,11 @@ class FMT_API format_error : public std::runtime_error {
   format_error& operator=(const format_error&) = default;
   format_error(format_error&&) = default;
   format_error& operator=(format_error&&) = default;
-  ~format_error() FMT_NOEXCEPT FMT_OVERRIDE;
+  ~format_error() FMT_NOEXCEPT FMT_OVERRIDE
+#if FMT_MSC_VER
+    = default
+#endif
```

Comment:
I suggest moving this into a separate macro `FMT_MSC_DEFAULT` that expands to `= default` if `FMT_MSC_VER` is set.

Reply:
Yeah, just my thought. I wasn't sure if a single occurence warrants another macro. But now that it's blessed I will change it as suggested tomorrow.